.github/workflows/test.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: CI on push release tag
+on:
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  run-tests:
+    name: Run DTS tests
+    if: ${{ github.head_ref == 'develop' }}
+    runs-on:
+      labels: dts-builder
+    steps:
+      - name: Checkout OSFV repo
+        shell: bash
+        run: |
+          git clone https://github.com/Dasharo/open-source-firmware-validation.git
+      - name: Install requirements
+        shell: bash
+        run: |
+          cd open-source-firmware-validation
+          git submodule update --init --checkout
+          python3 -m virtualenv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+      - name: Run QEMU
+        shell: bash
+        id: run_qemu
+        run: |
+          ./scripts/ci/qemu-run.sh nographic firmware &
+          pid=$!
+          echo "qemu_pid=$pid" >> "$GITHUB_OUTPUT"
+      # TODO: Add DPP keys to secrets and use them in tests
+      - name: Run tests
+        shell: bash
+        env:
+          DPP_EMAIL: ${{secrets.DPP_EMAIL}}
+          DPP_PASSWORD: ${{secrets.DPP_PASSWORD}}
+        run: |
+          robot -L TRACE -v config:qemu -v rte_ip:127.0.0.1 -v snipeit:no \
+             dts/dts-e2e.robot > /tmp/dts-test-ci
+      - name: Cleanup
+        shell: bash
+        run: |
+          deactivate
+          qemu_pid="${{ steps.run_qemu.outputs.qemu_pid }}"
+          kill $qemu_pid
+          cd ..
+          rm -rf open-source-firmware-validation
```

Comment:
Shouldn't we save logs somewhere so it can be debugged? And maybe path to logs on builder should be printed out?

Reply:
https://github.com/Dasharo/meta-dts/compare/df52aad37e5be1793811bae2603d8c3daec0688f..21a11dd5a691bfea6616bb097deeb37071100320